fe/lib/components/BrandSelect/BrandSelect.tsx

```diff
@@ -0,0 +1,130 @@
+import { ApolloClient, useQuery } from '@apollo/client';
+import {
+  FieldMessage,
+  Heading,
+  Inline,
+  Loader,
+  Secondary,
+  Stack,
+  Text,
+} from 'braid-design-system';
+import React, { useState } from 'react';
+
+import {
+  AdvertisementBrandingFieldsFragment,
+  AdvertisementBrandingsQuery,
+  AdvertisementBrandingsQueryVariables,
+  JobCategoriesQuery,
+} from '../../types/seekApi.graphql';
+import { JOB_CATEGORIES } from '../JobCategorySelect/queries';
+
+import { PaginatedBrands } from './PaginatedBrands';
+import { ADVERTISEMENT_BRANDINGS } from './queries';
+
+export interface BrandSelectProps {
+  client?: ApolloClient<unknown>;
+  hirerId: string;
+  errorMessage?: string;
+  initialBrandId?: string;
+  /**
+   * Type definition from the `Controller` component that's provided by `react-hook-form`.
+   */
+  onChange: (...event: any[]) => void;
+}
+
+export const BrandSelect = ({
+  client,
+  hirerId,
+  errorMessage,
+  onChange,
+  initialBrandId,
+}: BrandSelectProps) => {
+  const PAGE_SIZE = 4;
+
+  const [variables, setVariables] =
+    useState<AdvertisementBrandingsQueryVariables>({
+      hirerId,
+      first: PAGE_SIZE,
+    });
+
+  console.log('variables', variables);
+  const { data, previousData, loading } = useQuery<
+    AdvertisementBrandingsQuery,
+    AdvertisementBrandingsQueryVariables
+  >(ADVERTISEMENT_BRANDINGS, {
+    ...(client && { client }),
+    variables,
+  });
+
+  const { data: categoriesData } = useQuery<JobCategoriesQuery>(
+    JOB_CATEGORIES,
+    {
+      ...(client && { client }),
+      variables: {
+        schemeId: 'seekANZ',
+      },
+    },
+  );
```

Comment:
Was this just for testing? Doesn't seem to belong here.

Reply:
yeah, I tested it to see if it's mocked ok with categories, with the mocked apollo client, I should have cleaned it up